Sasquatch/Sasquatch/ViewControllers/MSMainViewController.swift

```diff
@@ -92,7 +93,7 @@ class MSMainViewController: UITableViewController, AppCenterProtocol, CLLocation
 
     // Miscellaneous section.
     self.installId.text = appCenter.installId()
-    self.appSecret.text = appCenter.appSecret()
+    self.appSecret.text = prodAppSecret()
```

Comment:
Revert your change. One Collector doesn't have app secret concept, you can just leave the app secret there.